statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -607,8 +674,8 @@ def smooth(self, params, transformed=True, complex_step=False,
                                   cov_kwds, results_class,
                                   results_wrapper_class)
 
-    _loglike_param_names = ['transformed', 'complex_step']
-    _loglike_param_defaults = [True, False]
+    _loglike_param_names = ['transformed', 'includes_fixed', 'complex_step']
+    _loglike_param_defaults = [True, False, False]
```

Comment:
would it make sense for _loglike_param_names and _loglike_param_defaults to be defined as a single dict?  especially if they are liable to grow, that'll make it harder to misread.

Reply:
Probably!